### ISSUE TITLE
VO Button + Space doesn't work on Safari with select2

### ISSUE THREAD

**Issue by bharadwajpro** [select2/select2#6021]
I am trying to activate the select2 by using Control+Option+Space (VO Button+Space), but it doesn't activate the select2 on Safari. Voiceover announces to press Control+Option+Space to activate it but pressing it doesn't activate it. Control+Option+Space works fine on Chrome. But the same keys doesn't activate on Safari. Only space and enter keys are activating it. We are experiencing this issue on our website as well. I am showing your website where the issue is present to make it easy to reproduce for select2 contributors.

Link to reproduce: https://select2.org/tagging
[![VO announces to press Control+Option+Space but it doesn't work on Safari][1]][1]

  [1]: https://i.stack.imgur.com/LgQa4.png

**Comment by kevin-brown** [comment]
I recognize that this is an issue but unfortunately I'm not able to test a fix for it. Any assistance or pull requests would be greatly appreciated and I'll gladly review them.
